src/Mobs/PassiveMonster.cpp

```diff
@@ -133,7 +138,7 @@ void cPassiveMonster::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 
 				virtual bool Item(cEntity * a_Entity) override
 				{
-					// if we're the same species as someone around and they dont have a partner, swipe right
+					// if we're the same species as someone around and they don't have a partner, swipe right.
```

Comment:
If I need to go visit urban dictionary to understand something, then it's not a good comment.